Formula/k/kubetui.rb

```diff
@@ -0,0 +1,40 @@
+class Kubetui < Formula
+  desc "TUI tool for monitoring and exploration of Kubernetes resources"
+  homepage "https://github.com/sarub0b0/kubetui"
+  url "https://github.com/sarub0b0/kubetui/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "137ef45cb539a56b16990c72ae60c7247d4ff90be3c16eec029dbd5e0480e677"
+  license "MIT"
+  head "https://github.com/sarub0b0/kubetui.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/kubetui --version")
+
+    cmd = "#{bin}/kubetui --kubeconfig not_exist 2>&1"
+    expected_error = "failed to read kubeconfig"
+
+    if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Let's use this flow on macOS too. There isn't really a reason not to

Reply:
I'll give it a try.